new_solution.py

```diff
@@ -70,25 +164,801 @@ def _convert_step1(self):
+                    ],
+                    fm_member   
+                )
+                if extract[i]["subjectRelation"].lower() == "інше" and \
+                   "relations_other" in fm_member:
+                   extract[i]["subjectRelation"] += \
+                                    ", " + fm_member["relations_other"]
+        elif raw_family_info:
+            for i, fm_member in enumerate(self._parse_raw_family_string(
+                                                raw_family_info)):
+                extract[i] = {}
+                extract[i]["raw"] = fm_member.get("raw", "")
+                extract[i]["subjectRelation"] = fm_member.get("relations", "")
+                extract[i]["bio_declomua"] = fm_member.get("family_name", "")
+        
+        return {
+            "step_2": extract
+        }
+
+    def _convert_step3(self):
+        extract = {}
+        record_counter = 1
+        estate_desc_dict = {
+            '23': 'Земельна ділянка',
+            '24': 'Житловий будинок',
+            '25': 'Квартира',
+            '26': 'Садовий (дачний) будинок',
+            '27': 'Гараж',
+            '28': 'Інше нерухоме майно',
+            '29': 'Земельна ділянка',
+            '30': 'Житловий будинок',
+            '31': 'Квартира',
+            '32': 'Садовий (дачний) будинок',
+            '33': 'Гараж',
+            '34': 'Інше нерухоме майно'
+        }
+        estate_info = self._jsrch("estate")
+        if estate_info:
+            for estate_id, estate_items in estate_info.items():
+                owner_id = ('1'
+                            if int(estate_id) in range(23, 29)
+                            else 'family')
+                for estate in estate_items:
+                    if 'space' in estate and estate['space']:
+                        extract[record_counter] = \
+                            self._convert_using_rules(
+                                [
+                                 (None, 'city', ""),
+                                 (None, 'person', ""),
+                                 ('region', 'place_oblast_declcomua', ""),
+                                 ('address', 'place_address_declcomua', ""),
+                                 (None, 'rights', {}),
+                                 (None, 'country', ""),
+                                 (None, 'cityPath', ""),
+                                 ('costs', 'costDate', ""),
+                                 (None, 'district', ""),
+                                 (None, 'postCode', ""),
+                                 (None, 'iteration', ""),
+                                 ('space', 'totalArea', ""),
+                                 (None, 'ua_street', ""),
+                                 (None, 'objectType', ""),
+                                 (None, 'owningDate', ""),
+                                 (None, 'objectType', ""),
+                                 (None, 'ua_cityType', ""),
```

Comment:
Please clarify it. In recent declarations I can see region set in ua_cityType field.

For example:
https://public-api.nazk.gov.ua/v1/declaration/43982943-477f-40ea-9dd2-57c834994819
https://declarations.com.ua/declaration/nacp_43982943-477f-40ea-9dd2-57c834994819